source/guides/account/scams-and-security.rst

```diff
@@ -0,0 +1,104 @@
+.. post:: 24 Feb, 2022
+    :category: Account
+    :tags: wallet
+    :excerpt: 1
+    :nocomments:
+
+######################
+Common Hacks and Scams
+######################
+
+***********
+Key Points
+***********
+
+- **Be extremely cautious signing transactions when activating delegated harvesting**. A Sweeping scam has actively targeted users in the process of activating delegated harvesting, so make sure to carefully review any transactions you are prompted to sign and diligently check for any included asset transfers to unfamiliar addresses. Never sign a transactions that you do not completely understand.
+- **Never share your private key or mnemonic phrase** (also known as your secret recovery phrase or seed phrase) with anyone, and never enter them on any website or potentially malicious wallet. Anyone with your private key or mnemonic phrase has full and unrestrained access to the contents of your wallet.  
+- **The Symbol Syndicate will never directly message you for customer support** and will never ask for your private key or mnemonic phrase. 
+- **Blockchain transactions are irreversible**; it is not feasible to undo a transaction or retrieve your funds once they are sent to a malicious address. 
+- **Ensure any backups of your private keys or mnemonic phrases are stored safely and offline** in order to ensure that no one is able to access them and compromise your accounts.
+
+*******************************************
+Sweeping with Aggregate Bonded Transactions
+*******************************************
+
+What is an Aggregate Bonded transaction?
+========================================
+
+Aggregate Bonded transactions can include a variety of elements such as multiple signers, messages, and transfers. New Symbol users may not be familiar with the complex nature of Aggregate Bonded transactions, and must take great care to fully understand the full scope of any transactions they are asked to sign. For more information see :doc:`Aggregate Transaction <../../concepts/aggregate-transaction>`
+
+
+What is sweeping?
+=================
+
+Sweeping involves a malicious script that monitors certain types of transactions, then initiates a malicious Aggregate Bonded transaction disguised to appear related to the targeted transaction (i.e. by including a relevant message), but which also include a transfer of XYM or other mosaics to the bad actor's wallet. Signing these transactions can completely empty a user's wallet, so please make sure you carefully review before signing any transactions that you did not initiate.
+
+It may not be immediately apparent that you've been hacked by a sweeper. If you assumed that the transaction you signed was part of a process you were completing, you may also assume that your balance has been reduced by a stuck transaction, pending activation of delegated harvesting, or that your balance was transferred to a delegated harvesting address linked to your account.
```

Comment:
```suggestion
It may not be immediately apparent that you've been hacked. The sweeper's goal is that you assume that the transaction you signed was part of a process you were completing, and you may also assume that your balance has been reduced by a stuck transaction, pending activation of delegated harvesting, or that your balance was transferred to a delegated harvesting address linked to your account.
```

Just to make extra clear that THAT is NOT what happened.